invenio_openaire/loaders.py

```diff
@@ -211,6 +215,46 @@ def grantxml2json(self, grant_xml):
         }
         return ret_json
 
+    def grantjson2json(self, grant_json):
+        """Convert OpenAIRE grant JSON into comptaible JSON."""
+        funder_doi = 0  # data is available, just have to be decided how it is fetched depending on the final integration
+        
+        to_replace = 0  # Code used to generate eurepo identifier
+
+        grant_id = grant_json["code"]
+        id_ = f"{funder_doi}::{grant_id}"
+
+        ret_json = {
+            "created": datetime.isoformat(datetime.now()),
+            "id": id_,
+            "links": {
+                "self": f"https://zenodo.org/api/grants/{id_}"
+            },
+            "metadata": {
```

Comment:
I think the content of the `metadata` key is the only data you need to return from the function ([see above](https://github.com/inveniosoftware/invenio-openaire/pull/65/files#diff-0b8daabf9d6e507efa6b23f0486c0bc543f8319ae5bc2cb86cdc825ef87a96b4R198))